db/migrate/20231116100955_add_currency_to_creditors.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrencyToCreditors < ActiveRecord::Migration[7.0]
+  def change
+    add_column :creditors, :currency, :text
```

Comment:
Consider modifying the model as well to set some expectations around this.
Is this meant to be a currency symbol like '$', or a currency code like 'USD'? Perhaps it should be validated.
Also, :text is a large storage format for either of those. Consider :string instead, and even a string limited in size to a handful of characters, whatever the max is for the specific value you are trying to store.

Reply:
Thank you for the feedback. Yes, it is meant to be a currency symbol. I will take all those into consideration,